docs/features/teams.rst

```diff
@@ -0,0 +1,85 @@
+.. _teams:
+
+Teams
+=====
+
+Pootle added the team page which is meant to ease management of team members
+and their roles within the team, and other team related aspects and features.
+The team page also provides overall statistics and access to the bulk handling
+for pending suggestions.
+
+.. image:: ../_static/accessing_team.png
+
+Language administrators will find the team interface on the first dropdown in
+their language's browse page.
+
+The team page is divided in two parts, on the left the management of the team
+members and on the right are listed the language's projects, and the overall
+translation statistics with links to the editor and the bulk suggestions
+manager.
+
+
+.. _teams#manage-members:
+
+Managing team members
+---------------------
+
+The UI for team members management is divided in two parts: on the upper side
+there is a form to add a new member, and below the team members are listed in
+four groups attending to their role within the group.
+
+
+.. _teams#roles:
+
+Roles
+^^^^^
+
+There are four different roles, which provide incremental permissions within
+the team:
+
+Member
+  This role gives the right to submit suggestions.
+
+Submitter
+  Provides permission to submit new translations and also suggestions.
+
+Reviewer
+  This roles allows to review suggestions within the team, and also submit new
+  translations and suggestions.
+
+Administrator
+  Allows to administer team members and their roles, as well as editing the
+  announcement for the team. It also allows to review, translate or suggest
+  translations.
+
+
+.. _teams#add-member:
+
+Adding a member
+^^^^^^^^^^^^^^^
+
+.. image:: ../_static/add_team_member.png
+
+New members can be added by using the form at the top of the team page. In
+order to do so just select the user, choose a role and click on **Add team
+member** button.
+
+
+.. _teams#remove-member:
+
+Removing a member
+^^^^^^^^^^^^^^^^^
+
+.. image:: ../_static/remove_team_member.png
+
+In order to remove members from the team just check the checkboxes besides
+their usernames, and then click on the **Remove selected** button. 
```

Comment:
To remove members check the box next to their username and then select "Remove selected"